src/lib/Welcome.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts">
+    import { openFolder } from "./File";
+    import { DocumentBlank, Folder, RecentlyViewed, Settings } from "carbon-icons-svelte";
+    import { addEditorTab, addTab } from "./EditorTabList.svelte";
+    import settings from "./Settings.svelte";
+    import { getVersion } from '@tauri-apps/api/app';
+    import { onMount } from "svelte";
+
+    export let hidden = true;
+
+    let appVersion = "";
+
+    onMount(async () => {
+        appVersion = await getVersion();
+    })
+</script>
+
+<div class="container" class:hidden>
+    <div class="content">
+        <div class="logo"></div>
+
+        <h1>Nucleus</h1>
+
+        <div class="quick-actions">
+            <button on:click={() => addEditorTab()} ><DocumentBlank /> New file</button>
+            <button on:click={openFolder} ><Folder /> Open Folder</button>
+            <!-- <button on:click={openFolder} ><RecentlyViewed /> Recent...</button> -->
+            <button on:click={() => addTab("Settings", "Settings", new settings({target: document.getElementById("tabview")}))}><Settings /> Config</button>
```

Comment:
Instead of "Config" it should be called "Settings" to keep the naming consistent